tests/virt/node/general/test_machinetype.py

```diff
@@ -35,155 +59,166 @@ def vm(request, cluster_cpu_model_scope_function, unprivileged_client, namespace
     [
         pytest.param(
             {"vm_name": "pc-q35", "machine_type": MachineTypesNames.pc_q35_rhel7_6},
             MachineTypesNames.pc_q35_rhel7_6,
             marks=pytest.mark.polarion("CNV-3311"),
         )
     ],
-    indirect=["vm"],
+    indirect=["vm_for_machine_type_test"],
 )
-def test_pc_q35_vm_machine_type(vm, expected):
-    validate_machine_type(vm=vm, expected_machine_type=expected)
-
-
-@pytest.mark.parametrize(
-    "vm",
-    [
-        pytest.param(
-            {"vm_name": "machine-type-mig"},
-            marks=pytest.mark.polarion("CNV-3323"),
-        )
-    ],
-    indirect=True,
-)
-@pytest.mark.arm64
-@pytest.mark.rwx_default_storage
-@pytest.mark.gating
-@pytest.mark.conformance
-def test_migrate_vm(machine_type_from_kubevirt_config, vm):
-    migrate_vm_and_verify(vm=vm)
-
-    validate_machine_type(vm=vm, expected_machine_type=machine_type_from_kubevirt_config)
+def test_pc_q35_vm_machine_type(vm_for_machine_type_test, expected):
+    validate_machine_type(vm=vm_for_machine_type_test, expected_machine_type=expected)
 
 
 @pytest.mark.parametrize(
-    "vm, updated_kubevirt_config_machine_type",
+    "vm_for_machine_type_test",
     [
         pytest.param(
             {"vm_name": "default-kubevirt-config"},
-            {"machine_type": MachineTypesNames.pc_q35_rhel8_1},
-            marks=pytest.mark.polarion("CNV-4347"),
         )
     ],
     indirect=True,
 )
+@pytest.mark.usefixtures("vm_for_machine_type_test")
 @pytest.mark.gating
-@pytest.mark.conformance
-@pytest.mark.usefixtures("xfail_if_jira_75031_is_open")
-def test_machine_type_after_vm_restart(
-    machine_type_from_kubevirt_config,
-    vm,
-    updated_kubevirt_config_machine_type,
-    restarted_vm,
-):
-    """Test machine type change in kubevirt_config; existing VM does not get new
-    value after restart"""
-    validate_machine_type(vm=vm, expected_machine_type=machine_type_from_kubevirt_config)
-
+class TestMachineType:
+    @pytest.mark.arm64
+    @pytest.mark.gating
+    @pytest.mark.conformance
+    @pytest.mark.polarion("CNV-5658")
+    def test_major_release_machine_type(self, machine_type_from_kubevirt_config):
+        # CNV should always use a major release for machine type, for example: pc-q35-rhel9.6.0
+        assert machine_type_from_kubevirt_config.endswith(".0"), (
+            f"Machine type should be a major release {machine_type_from_kubevirt_config}"
+        )
 
-@pytest.mark.parametrize(
-    "vm, updated_kubevirt_config_machine_type",
-    [
-        pytest.param(
-            {"vm_name": "default-kubevirt-config"},
-            {"machine_type": MachineTypesNames.pc_q35_rhel8_1},
-            marks=pytest.mark.polarion("CNV-11268"),
+    @pytest.mark.conformance
+    @pytest.mark.polarion("CNV-8561")
+    def test_machine_type_as_rhel_9_6(self, machine_type_from_kubevirt_config):
+        """Verify that machine type in KubeVirt CR match the value pc-q35-rhel9.6.0"""
```

Comment:
1. why hard code this? iirc the value is based on the nodes OS, right?
2. what is going to happen on a imxed rhel cluster? will it still be rhel 9.6?

Reply:
we run this test only in x86_64 and pc-q35-rhel9.6.0 is hardcoded default in Kubevirt now

Reply:
It was already present, I'm not adding it

It is a good candidate to be moved to tier1, but until we have something there - we need to keep it in tier2
I think we will have a big task for moving all non-relevant tests to tier1